docs/README.md

```diff
@@ -0,0 +1,29 @@
+# Microsoft Graph SDK Docs
+
+## What is Microsoft Graph?
+Microsoft Graph is a unified endpoint for accessing data, intelligence, and insights coming from the Microsoft cloud. You can find more information at [developer.microsoft.com/graph](https://developer.microsoft.com/graph)
+
+## Why an SDK?
+Users can begin developing immediately by connecting with the REST API directly if they desire. If you are interested in diving right in to Graph data, the SDK provides a few shortcuts:
+
+- HTTP logic is written for you so you reduce the amount of boilerplate you need to get started
+- The SDK gives you access to PHP objects that you can use to access and update instead of working directly with JSON
+- An extra layer of error handling and intelligent defaults helps you debug your application
+
+## Get started
+View the [getting started steps](https://github.com/microsoftgraph/msgraph-sdk-php#get-started-with-microsoft-graph) to make your first request with Graph.
+
+## Using the SDK
+- [Usage Examples](https://github.com/microsoftgraph/msgraph-sdk-php/tree/dev/docs/Examples.md)
+- [Documentation](https://github.com/microsoftgraph/msgraph-sdk-php/tree/dev/docs/index.html)
+
+To make calls to the Beta Microsoft Graph API, check out our [Microsoft Graph Beta SDK](https://packagist.org/packages/microsoft/microsoft-graph-beta).
+## Support
+View the [contributions guidelines](https://github.com/microsoftgraph/msgraph-sdk-php/blob/main/CONTRIBUTING.md) for more information on submitting a pull request.
+
+Check out our [Frequently Asked Questions](https://github.com/microsoftgraph/msgraph-sdk-php/tree/dev/docs/FAQ.md).
+
+Bugs are filed and monitored in [GitHub issues](https://github.com/microsoftgraph/msgraph-sdk-php/issues).
+
+You can also ask developers and the community Graph-related questions on [Stack Overflow](https://stackoverflow.com/questions/tagged/microsoftgraph?sort=newest)

```

Comment:
We should point customers now to Microsoft Q&A.

https://docs.microsoft.com/en-us/answers/topics/128347/microsoft-graph-sdk.html